aws_marketplace/using_model_packages/preprocessing-audio-data-using-a-machine-learning-model/preprocessing-audio-data-using-a-machine-learning-model.ipynb

```diff
@@ -0,0 +1,894 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !wget -O dev_data_pump.zip https://zenodo.org/record/3678171/files/dev_data_pump.zip?download=1
</pre><p>It's preferred for AWS to host a copy of the dataset, since this 3rd-party data could be moved or deleted at any time, which would break the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3061/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Confirmed we will leave the dataset source as-is.</p>